package.json

```diff
@@ -111,7 +111,7 @@
     "classlist.js": "1.1.20150312",
     "core-js": "3.7.0",
     "focus-trap": "6.1.4",
-    "fundamental-styles": "0.17.1-rc.2",
+    "fundamental-styles": "0.18.0-rc.0",
```

Comment:
@Lokanathan-k since you are bringing the next version (most probably it brings a breaking change) so you need to make sure you don't break other component

Reply:
Hi @droshev from `0.17.1-rc.2` until `0.18.0-rc.0`,  most of the changes brought in were library updates to `package.json` in Styles. 
Other than this, 2 PRs dealt with a new User Menu component(which is not yet introduced in ngx), and one with few file uploader changes; both of which do not break the components.

Reply:
someone has to bring https://github.com/SAP/fundamental-styles/wiki/Breaking-Changes#0180

Reply:
@olshansky has already brought in some of these changes to PR https://github.com/SAP/fundamental-ngx/pull/5074. Also noticed that this PR is also cleaning up styles as well. To reduce duplicate effort, I think I can close this PR in favour of PR 5074. 
Please let me know if that is okay @droshev